### ISSUE TITLE
Update column family names to match Zebra's database design

### ISSUE THREAD

**Issue by teor2345** [ZcashFoundation/zebra#4150]
## Motivation

To speed up development and minimise diffs, we kept the same column family names in Zebra's database.

But we should update them to match the design. This requires a state version increment.

### Designs

The column family names should match:
https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/rfcs/0005-state-updates.md#rocksdb-data-structures

**Comment by teor2345** [comment]
PR #4355 contains another state version increment which fixes a bug.

**Comment by upbqdn** [comment]
This issue is resolved by PR #4639. I used the string `Addresses #4150` in the description of the PR, but GitHub apparently didn't recognize it and didn't automatically close this issue. It turns out one has to use one of these keywords in order to trigger the automatic closing:
```
close, closes, closed, fix, fixes, fixed, resolve, resolves, resolved
```
